src/stylesheets/components/_toolbar.scss

```diff
@@ -163,4 +163,194 @@
   margin-left: 20px;
 }
 
+.sds-accordion__trigger {
+  display: block;
+  font-size: 17px;
+  font-weight: 600;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  text-align: left;
+  width: 100%;
+  outline: none !important;
+  cursor: pointer;
+  border-style: none;
+  background-color: white;
+}
+
+.sds-accordion__trigger:focus,
+.sds-accordion__trigger:hover {
+  background: #eff6fb;
+}
+
+.sds-accordion .sds-accordion__trigger::-moz-focus-inner {
+  border: 0;
+}
+
+.sds-accordion__title {
+  display: block;
+  pointer-events: none;
+  border: transparent 1px dashed;
+  border-radius: 2px;
+  outline: none;
+  padding: 6px 10px;
```

Comment:
Can we update these to units?

I used this for the matching filter panel (4px and 8px):
padding: units('05') units(1);

Reply:
toolbar component